clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy

```diff
@@ -169,6 +169,7 @@ class OperationsController {
         )
         if (validator) {
           validator.validate(descriptions, description, errors)
```

Comment:
I don't think it matters in this case. In fact, an optimization could be to not authorize at all if `errors` already contains errors, but I think a better developer experience includes everything wrong with the request at once, rather than try 1 thing, fix the error, try it again, fix a different error, etc.